Cargo.lock

```diff
@@ -8096,7 +8095,7 @@ version = "1.6.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "4ee73e6e4924fe940354b8d4d98cad5231175d615cd855b758adc658c0aac6a0"
 dependencies = [
- "cfg-if 1.0.0",
+ "cfg-if 0.1.10",
```

Comment:
I don't know why this changed :thinking:

Reply:
I've been faced with such behavior after running `cargo update -p ...` sometimes. 

Maybe `cargo update -p cfg-if` will help?

Reply:
`cargo update -p cfg-if:0.1.10`reverted these changes but as I see we have both of them (v0.1.10 and v1.0.0) as indirect dependencies. Therefore it seems to be not a problem.